db/migrate/20160607141810_remove_database_synchronization_role.rb

```diff
@@ -0,0 +1,25 @@
+class RemoveDatabaseSynchronizationRole < ActiveRecord::Migration[5.0]
+  class ServerRole < ActiveRecord::Base; end
+  class AssignedServerRole < ActiveRecord::Base; end
+  class SettingsChange < ActiveRecord::Base
+    serialize :value
+  end
+
+  def up
+    say_with_time("Removing database synchronization role") do
+      role = ServerRole.where(:name => "database_synchronization").first
+      if role
+        AssignedServerRole.delete_all(:server_role_id => role.id)
+        role.delete
+      end
+    end
+
+    say_with_time("Removing role from currently configured servers") do
+      changes = SettingsChange.where(:key => "/server/role")
+      changes.each do |change|
+        change.value.delete("database_synchronization")
+        change.save!
```

Comment:
role is a comma separated list I believe, so does this work?  Do you have a test where database_synchronization is in the middle of the list?